design/vela-core/migrate-from-oam-runtime.md

```diff
@@ -0,0 +1,154 @@
+# How to upgrade to KubeVela
+
+What if I want to upgrade from [oam-kubernetes-runtime](https://github.com/crossplane/oam-kubernetes-runtime) to KubeVela? Here's a detailed guide!
+has already been merged into [vela-core](https://github.com/oam-dev/kubevela/pull/663) now. Here is the doc for users who want
+to migrate from OAM Runtime to use KubeVela.
+
+## For Users Who are Using OAM runtime as Standalone Controller
+
+If you are using OAM Runtime as Standalone Controller, migrating to KubeVela is very straight forward.
+
+Server-side KubeVela(We call it `vela-core` for convenience) now includes following **BUILT-IN** CRDs and controllers.
+
+| Type |  CRD   | Controller  | From |
+| ---- |  ----  | ----  | ----  |
+| Control Plane Object | `applicationconfigurations.core.oam.dev` | Yes | OAM Runtime |
+| Control Plane Object | `components.core.oam.dev` | Yes | OAM Runtime |
+| Workload | `containerizedworklaods.core.oam.dev` | Yes | OAM Runtime |
+| Scope | `healthscope.core.oam.dev` | Yes | OAM Runtime |
+| Trait | `manualscalertraits.core.oam.dev` | Yes | OAM Runtime |
+| Control Plane Object | `scopedefinitions.core.oam.dev` | No | OAM Runtime |
+| Control Plane Object | `traitdefinitions.core.oam.dev` | No | OAM Runtime |
+| Control Plane Object | `workloaddefinitions.core.oam.dev` | No | OAM Runtime |
+| Trait | `autoscalers.standard.oam.dev` | Yes | New in KubeVela |
+| Trait | `metricstraits.standard.oam.dev` | Yes | New in KubeVela |
+| Workload | `podspecworkloads.standard.oam.dev` | Yes | New in KubeVela |
+| Trait | `route.standard.oam.dev` | Yes | New in KubeVela |
+
+CRDs and Controllers in the table from 'OAM Runtime' are exactly the same to those in `oam-kubernetes-runtime`.
+So in KubeVela we have added 4 more new CRDs with controller. 
+
+### Option 1: You want only Pure OAM Runtime by using vela-core with No additional traits and workloads.
+
+1. Find you deployment
+
+```shell script
+$ kubectl -n oam-system get deployment -l app.kubernetes.io/name=oam-kubernetes-runtime
+NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
+oam-kubernetes-runtime-oam   1/1     1            1           62s
+```
+
+2. Update the deployment
```

Comment:
Why not just delete OAM Runtime deployment and install Vela server deployment?
This would not affect any existing applications any way.

Reply:
Update these two lines is easiest